framework/sandbox/icons/app.tsx

```diff
@@ -1,26 +1,63 @@
 import React from 'react'
-import { NavLink } from 'react-router-dom'
-import { Code, Divider, H1, Link, P, Stack, Text } from '../../lib'
+import { Link as NavLink } from 'react-router-dom'
+import { BrightnessDuo, BrightnessSolid, SmileyDuo } from '@northlight/icons'
+import { Card, CardBody, CardFooter, CardHeader, Center, Code, Divider, H1, H3, Heading, Icon, Link, P, SimpleGrid, Spacer, Stack, Text } from '../../lib'
 
 const Icons = () => (
   <Stack p={ 4 } spacing={ 4 } bgColor="background.default" height="auto">
-    <H1>Mediatool icons</H1>
-    <P>Exported as components from the package{ ' ' }
+    <H1>Icons <Icon as={ SmileyDuo } sx={ { color: 'blue.500' } } /> </H1>
+    <P>Northlight provides a set of commonly used interface icons you can use in your project, exported as components from the package{ ' ' }
       <Code>@northlight/icons</Code>
     </P>
-    <Divider />
-    <Stack>
-      <P>Mediatool uses 2 variants of icons: { ' ' }
+
+    <Center>
+      <H3 mt={ 5 } mb={ 5 }>Select Variant</H3>
+    </Center>
+    <SimpleGrid spacing={ 10 } alignItems="center" minChildWidth="300px" templateColumns="repeat(auto-fit, minmax(200px, 1fr))">
+
+      <Spacer />
+      <Link as={ NavLink } sx={ { borderRadius: 'lg', _hover: { textDecoration: 'none', bgColor: 'blue.500', color: 'gray.50' } } } to="/icons/duo">
+        <Card align="center" variant="outline" sx={ { _hover: { color: 'mono.white', borderColor: 'blue.500' } } }>
```

Comment:
Setting the color to mono.white on hover makes it so the color of the icon becomes white as the color is inherited, which makes it invisible when hovering